### DISCUSSION TITLE
Franka Reach Sim-to-Real

### DISCUSSION THREAD

**Discussion by sesem738** [isaac-sim/IsaacLab#1355] [Q&A] [answered]
I have successfully trained a Reach task policy for the Franka Research 3 using RL. As this is my first time working with the Franka hardware, I noticed it primarily uses C++ for control. Are there any Python wrappers available that would allow me to deploy my trained RL policy on the physical robot?

**Comment by RandomOakForest** [comment, marked as answer]
I'm afraid not for this case specifically. Please, see if our documentation on [Additional Resources](https://isaac-sim.github.io/IsaacLab/main/source/refs/additional_resources.html) is of any help. Also, you may be interested in [deploying policies in Isaac Sim](https://docs.omniverse.nvidia.com/isaacsim/latest/isaac_lab_tutorials/tutorial_policy_deployment.html).
